recipes/containerpilot/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "containerpilot" %}
+{% set version = "2.7.8" %}
+{% set sha1= "09158be44c3e887244581d4d019748df9fcfa93c" %}
```

Comment:
Using sha1 because that is what is available from the download site.

Reply:
You can download the tarball and compute the sha256 yourself, though....

Reply:
Sure, but isn't it safer and more efficient to use the one published by in the download site?

Reply:
Not really.  It doesn't really matter who computes the original hash as long as it is correct.